### PR TITLE
LPS-197809 - Accessibility: Data Set Fragment - Form elements must have an attribute

### PR DESCRIPTION
**Pull request by Youness-ELMO**
References
[LPS-197809 : Accessibility: Data Set Fragment - Form elements must have labels](https://liferay.atlassian.net/browse/LPS-197809)

What is the goal of this PR?
The goal of this PR is to add an attribute to the input to fix the accessibility issue.

### **Before PR**
image

### **After PR**
image

### CODE REVIEW

**Comment by Youness-ELMO** [comment]
ci:test:sf

**Comment by Youness-ELMO** [comment]
ci:test:sf

**Review by markocikos — CHANGES_REQUESTED** [review]
@Youness-ELMO Please see comment inline

**Comment by markocikos** [comment]
Github ate my comment :sweat_smile:

**Review by markocikos — CHANGES_REQUESTED** [review]
@Youness-ELMO Second try, please see comment inline.

Also, to fix or detect SF errors, please run `gradle formatSource` on the module.

File: modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/FiltersDropdown.js [hunk 1]
```diff
@@ -75,7 +75,12 @@ const FiltersDropdown = () => {
 				</>
 			) : (
 				<ClayDropDown.Group header={Liferay.Language.get('filters')}>
-					<ClayDropDown.Search onChange={onSearch} value={query} />
+					<ClayDropDown.Search
+						aria-label={Liferay.Language.get('input-filter')}
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/FiltersDropdown.js:79]: `input-filter` does not exist in [DXP language keys](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties). You need to add the key there if you want to use it. But, if you did this, `Input Filter` would be rather technical and non-descriptive label for screen reader to read. Something like `search` exists and would be more descriptive.

**Comment by Youness-ELMO** [comment]
ci:test:sf

**Review by markocikos — APPROVED** [review]
@Youness-ELMO LGTM! :sparkles:

**Comment by Youness-ELMO** [comment]
ci:forward

**Comment by markocikos** [comment]
ci:stop

**Comment by markocikos** [comment]
@Youness-ELMO Please don't do force forward, unless QA manually ckecks PR and finds errors to be unrelated.

**Comment by markocikos** [comment]
These error may actually be related. `packageRunTest` errors typically mean unit test or SF errors, and these are rarely flaky. Since we are changing markup, it may be that some unit ot snapshot tests in reported modules got broken and need to be updated.

**Comment by Youness-ELMO** [comment]
oh my bad, thanks for the clarification!

**Comment by markocikos** [comment]
ci:forward

**Pull request closed without merging**